traits/src/budget.rs

```diff
@@ -9,33 +9,47 @@ pub trait Budget {
 	/// Implementations should use interior mutabilitiy
 	fn consume_custom(&self, calls: u32) -> bool;
 
-	fn val(&self) -> u32 {
-		self.get_value()
+	fn budget_left_value(&self) -> u32 {
+		self.get_budget_left_value()
```

Comment:
A better name for this may be `budget_remaining_value`.

Reply:
I've seen cases for both namings. We currently use it in production for Phala, but I can make the change if we want to use `remaing` vs `left`

Reply:
If we already use it in production for Phala it is probably better to keep it then.

Reply:
@ilionic can weigh-in on this. The change isn't too major, so I think it should be okay if there is a preference and we will make updates through runtime upgrade in the future.